packages/rule-https-only/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "ava": {
+    "failFast": false,
+    "files": [
+      "dist/tests/**/*.js"
+    ],
+    "timeout": "1m"
+  },
+  "description": "sonarwhal rule that checks if your site is using HTTPS and if it has mixed content.",
+  "devDependencies": {
+    "@types/debug": "0.0.30",
```

Comment:
Do we need the `debug` types? Aren't they pulled from the main?

Reply:
I will take a look

Reply:
The `package.json` in main doesn't include `@types/debug`